ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2633,6 +2633,11 @@ router_bgp:
       local_as: < bgp_as >
       description: "< description as string >"
       shutdown: < true | false >
+      remove_private_as:
```

Comment:
The EOS CLI mentions `outbound` for the regular mode and inbound for the `ingress` keyword.
Remember that `neighbor 1.2.3.4 remove-private-as all replace-as` is also valid CLI - not covered in Petr's example.